### ISSUE TITLE
Installation issues with hatch

### ISSUE THREAD

**Issue by willtownes** [willtownes/spatial-factorization-py#5]
I received this via email from @jcai-1122 

I am trying to apply the developed python library to my data following instruction on this website: https://github.com/willtownes/spatial-factorization-py, but keeps encountering error report after I implement the last line 'hatch build' - please see attached screenshot below. Could you please give me a hint on how I could address this?

<img width="876" alt="Screenshot 2024-11-28 at 4 31 18 PM" src="https://github.com/user-attachments/assets/5307a90d-29d3-4d50-8981-135efe2d80d6">

**Comment by willtownes** [comment]
@jcai-1122 I just tried reinstalling everything and it works for me. You don't need to use hatch. Just follow the updated instructions in the readme file. Note that you need to use python 3.11 because tensorflow isn't supported on more recent python versions. Let me know if you have further difficulties.

**Comment by alam-shahul** [comment]
Yeah, `hatch` is just used for building the package and creating a binary for installation (for example, if you want to upload to PyPI, or if you just want different versions of the package to be available as separate binary files). This is mostly relevant for dev purposes, not for users.

Also, it looks like the error in the above message is because `hatch build` is being run in the wrong directory (need to `cd` into the project directory first).

**Comment by jcai-1122** [comment]
Thank you both for the detailed explanation! I will try it again!

Best,

Jiachen

From: alam-shahul ***@***.***>
Date: Friday, 6 December 2024 at 00:38
To: willtownes/spatial-factorization-py ***@***.***>
Cc: Cai, Jiachen ***@***.***>, Mention ***@***.***>
Subject: Re: [willtownes/spatial-factorization-py] Installation issues with hatch (Issue #5)
You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

Yeah, hatch is just used for building the package and creating a binary for installation (for example, if you want to upload to PyPI, or if you just want different versions of the package to be available as separate binary files). This is mostly relevant for dev purposes, not for users.

Also, it looks like the error in the above message is because hatch build is being run in the wrong directory (need to cd into the project directory first).
